policy/memory-resource_test.go

```
@@ -0,0 +1,645 @@
+		}
+		if r.Type == ResourceARNAll {
+			t.Errorf("ParseResource(%q) = wildcard; a bare prefix names no resource", prefix)
+		}
+		if !r.IsBareARN() {
+			t.Errorf("ParseResource(%q) did not mark the resource bare", prefix)
+		}
+		if r.Pattern != "" {
+			t.Errorf("ParseResource(%q): pattern = %q, want empty", prefix, r.Pattern)
+		}
+
+		// Inert: it matches nothing, which is the behaviour already on disk.
```

golangci-lint (2.12.2)

[error] `behaviour` is a misspelling of `behavior`

(misspell)